src/widgets/dragdrop.js

```diff
@@ -122,25 +121,51 @@ filepicker.extend('dragdrop', function(){
                 }
             }
 
-            var files = e.dataTransfer.files;
-            var total = files.length;
-            if (verifyUpload(files)) {
-                onStart(files);
-                //disabling
-                div.setAttribute('disabled', 'disabled');
-                for (i = 0; i < files.length; i++) {
-                    fp.store(files[i], store_options, getSuccessHandler(i, total), errorHandler, getProgressHandler(i, total));
+            var files = e.dataTransfer.files,
```

Comment:
Function `div.addEventListener('drop', ...` became too long. Please split it to some more functions for better documentation and code-following.